molgenis-metadata-manager/src/main/frontend/src/components/generic-components/AttributeTreeNode.vue

```diff
@@ -39,7 +39,7 @@
     },
     computed: {
       isFolder: function () {
-        return this.attribute.type === 'COMPOUND'
+        return this.attribute.type === 'compound'
```

Comment:
huh? When the attribute types get retrieved from the server, they are toUpperCased() before being put in the state....